src/Application/Model/Ticket.php

```diff
@@ -633,6 +633,17 @@ public function getDependingEncodingTicketState() {
 			
 			return $handle->fetch()['ticket_depending_encoding_ticket_state'];
 		}
+
+		public function isDependingEncodingTicketStateSatisfied() {
+			// break through model layer and access project attribute directly
```

Comment:
Maybe clarify that this is related to the use of a database function? *break through* seemed missleading on the first read.